src/loaders/spz/SpzLoader.js

```diff
@@ -355,8 +355,11 @@ export class SpzLoader {
 
     static loadFromURL(fileName, onProgress, minimumAlpha, compressionLevel, optimizeSplatData = true,
                        outSphericalHarmonicsDegree = 0, headers, sectionSize, sceneCenter, blockSize, bucketSize) {
+        const localOnProgress = (percent, percentLabel) => {
```

Comment:
I'm unsure of the possible side effect there, but this change at least obscures how the `onProgress` eventing functions, which already is not trivial to grasp.

I even doubt this change is necessary.